phpBB/install/app.php

```diff
@@ -20,9 +20,9 @@
 $phpbb_root_path = '../';
 $phpEx = substr(strrchr(__FILE__, '.'), 1);
 
-if (version_compare(PHP_VERSION, '7.1') < 0)
+if (version_compare(PHP_VERSION, '7.1.3') < 0)
```

Comment:
Is this check needed?, this file already load the `install/startup.php` that load `includes/startup.php` that already have that check

Reply:
The check in `includes/startup.php` is for production env., this one is for `installer` env.